Foundation/NSString.swift

```diff
@@ -1286,11 +1288,13 @@ extension NSString {
             throw NSError(domain: NSCocoaErrorDomain, code: CocoaError.fileReadInapplicableStringEncoding.rawValue, userInfo: [
                 "NSDebugDescription" : "Unable to bridge CFString to String."
                 ])
-        }    
+        }
+        
+        enc?.pointee = encoding
```

Comment:
Previously the code set `enc?.pointee` even if the `CFStringCreateWithBytes()` called failed and the function threw.  Now it doesn't.  Did you check what the behaviour of Darwin is in this regard?